Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -365,12 +408,22 @@ namespace O3DE::ProjectManager
 
     void CreateGem::HandleBackButton()
     {
-        if (m_tabWidget->currentIndex() > 0)
+        if (m_stackWidget->currentIndex() > gemTemplateSelectionScreen)
```

Comment:
The nice thing about not using the constants is the logic won't break if you re-order the tabs or add new ones.

Reply:
hmm yea, maybe I should change this one back to just `0` right?

Reply:
it's a minor thing but I would recommend it